http/basic/src/main/java/org/wildfly/security/http/basic/BasicAuthenticationMechanism.java

```diff
@@ -60,6 +62,7 @@
 
     private final boolean includeCharset;
     private final String configuredRealm;
+    private final Logger log = Logger.getLogger(BasicAuthenticationMechanism.class);
```

Comment:
@vijay-boddupalli just a minor. If you notice, we do already log some messages in this class. We have imported `httpBasic` logger at the beggining of this file. It would be better to use that one.

Reply:
Oh I must have missed that. Sure, I will go ahead and use that one then. Thanks.

Reply:
Looks like Diana and I added the same comment at the same time :)

Reply:
Fixed this one and pushed my changes